src/components/Aside/Aside.js

```diff
@@ -1,7 +1,36 @@
 import React from "react";
-
-const Aside = () => {
-  return <div>Aside</div>;
+import NavSideCategoryComponents from "../Nav/NavSideCategoryComponents";
+import "./Aside.scss";
+const Aside = ({ categoryList, powerOn, handleSideBarOn }) => {
+  return (
+    <aside className={`${powerOn === true ? "gsb" : "gsb hidden"}`}>
```

Comment:
```suggestion
    <aside className={powerOn === true ? "gsb" : "gsb hidden"}>
```

template literal 없어도 됩니다